### PR TITLE
Fix coverity scan issues

### PR DESCRIPTION
**Pull request by disaykin**

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by disaykin** [comment]
@tklengyel Something broke on the CI server?

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/libinjector/win_injector.c [hunk 1]
```diff
@@ -1660,26 +1665,34 @@ static event_response_t injector_int3_cb(drakvuf_t drakvuf, drakvuf_trap_info_t*
             return 0;
         }
 
-        access_context_t ctx = { 0 };
```
Comment (tklengyel) [inline src/libinjector/win_injector.c:1663]: These were specifically added to initialize the whole ctx structure to 0. Why are you removing it?

File: src/plugins/memdump/memdump.cpp [hunk 2]
```diff
@@ -428,7 +428,7 @@ bool inspect_stack_ptr(drakvuf_t drakvuf, drakvuf_trap_info_t* info, memdump* pl
     // read up to 512 bytes of stack, this may fail returning a partial result
     // thus, the following for loop analyzes the buffer only up to the `bytes_read` value
 
-    vmi_read(vmi, &ctx, 512, buf, &bytes_read);
+    (void)vmi_read(vmi, &ctx, 512, buf, &bytes_read);
```
Comment (tklengyel) [inline src/plugins/memdump/memdump.cpp:431]: The real fix would be to check the return value.

File: src/plugins/objmon/objmon.cpp [hunk 3]
```diff
@@ -150,27 +150,20 @@ static event_response_t cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info)
 {
 
     objmon* o = (objmon*)info->trap->data;
-    vmi_instance_t vmi = drakvuf_lock_and_get_vmi(drakvuf);
     struct ckey ckey = {};
 
-    access_context_t ctx;
-    memset(&ctx, 0, sizeof(access_context_t));
```
Comment (tklengyel) [inline src/plugins/objmon/objmon.cpp:157]: Why are you removing the memset?

File: src/libinjector/win_injector.c (hunk 1 above)
Comment (disaykin) [reply src/libinjector/win_injector.c:1663]: This is a designated initializer:

struct_t s = { .a = 2, .b = 3 };

Other members are initialized as zero: "Omitted field members are implicitly initialized the same as objects that have static storage duration." (https://gcc.gnu.org/onlinedocs/gcc/Designated-Inits.html)

So compiler do this for us.

File: src/plugins/memdump/memdump.cpp (hunk 2 above)
Comment (disaykin) [reply src/plugins/memdump/memdump.cpp:431]: The comment above indicates that a read error, or partial reading, is normal in this case.

File: src/plugins/objmon/objmon.cpp (hunk 3 above)
Comment (disaykin) [reply src/plugins/objmon/objmon.cpp:157]: Because of the designated initializer do this already.

File: src/plugins/memdump/memdump.cpp (hunk 2 above)
Comment (tklengyel) [reply src/plugins/memdump/memdump.cpp:431]: Ah yes, I missed that

**Pull request merged**